tensorbay/opendataset/CCPDGreen/loader.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name, missing-module-docstring
+
+import os
+from typing import Iterator
+
+from ...dataset import Data, Dataset
+from .._utility.glob import glob
+from ..CCPD.loader import get_label
+
+_CCPD_SEGMENTS = ("train", "val", "test")
+
+DATASET_NAME = "CCPDGreen"
+
+
+def CCPDGreen(path: str) -> Dataset:
+    """Dataloader of CCPDGreen open dataset.
+
+    .. _CCPD: https://github.com/detectRecog/CCPD
+
+    The file structure should be like::
+
+        <path>
+            CCPD2020/
+                ccpd_green/
+                    train/
+                    test/
+                    val/
+
+    Arguments:
+        path: The root directory of the dataset.
+
+    Returns:
+        Loaded :class: `~tensorbay.dataset.dataset.Dataset` instance.
+
+    """
+    root_path = os.path.abspath(os.path.expanduser(path))
+
+    dataset = Dataset(DATASET_NAME)
+    dataset.load_catalog(os.path.join(os.path.dirname(__file__), "catalog.json"))
+
+    for segment_tail in _CCPD_SEGMENTS:
+        segment = dataset.create_segment(f"green-{segment_tail}")
```

Comment:
Is `green-` prefix necessary?

Reply:
no, I just want to emphasize that they are from CCPDGreen.